src/lib/marshal/translation/helpers.ts

```diff
@@ -116,6 +117,7 @@ type CommandTargetProps = {
   flags: {
     all: boolean | undefined;
     "translations-dir": DirContext | undefined;
+    format?: string | undefined;
```

Comment:
Not sure if this needs to be optional, because the --format flag will always have a default value?

Reply:
`type CommandTargetProps` is used in `ensureValidCommandTarget` which is used by all of the translation commands, so it won't be defined for push/validate until we implement those.

Reply:
Ah gotcha, makes sense.